### PR TITLE
Add decrease tests for oracle token

### PR DESCRIPTION
**Pull request by kareem-ib**

### CODE REVIEW

**Comment by skmgoldin** [comment]
Hi Kareem, thank you for this excellent pull request. Could you make these changes before we merge it?

1. Your test `it('Should increase supply by 10 then decrease by 4'` is misspecified. It should say `it('Should increase supply by 10 then decrease by 6'`
2. In your test `it('Should increase the supply by 10 and fail to decrease it by 11'`, please use the `asOracle` function idiomatically.
3. Could you use `increaseAmount` and `decreaseAmount` like in the existing tests?

Thanks Kareem! Can't wait to merge this!

**Comment by kareem-ib** [comment]
Hey Mike! All the problems are fixed now (I hope)

File: test/eip621OraclizedToken.js [hunk 1]
```diff
@@ -41,27 +41,46 @@ contract(`EIP621OraclizedToken`, (accounts) => {
...
-        await asOracle(instance.decreaseSupply, amountDecrease, from)
+        await asOracle(instance.increaseSupply, increaseAmount, from)
+        await asOracle(instance.decreaseSupply, decreaseAmount, from)
         
         const recipientBalance = instance.balanceOf.call(from)
         const expectedBalance = amountFinal.toString();
 
         const totalSupply = instance.totalSupply.call()
-        const expectedSupply = (initialAmount + (amountIncrease - amountDecrease)).toString()
+        const expectedSupply = (initialAmount + (increaseAmount - decreaseAmount)).toString()
```
Comment (skmgoldin) [inline test/eip621OraclizedToken.js]: You already computed `increaseAmount - decreaseAmount` on line 47, so just use `amountFinal` here.

**Review by skmgoldin — CHANGES_REQUESTED** [review]
Just one change, then I'll merge!

**Review by skmgoldin — APPROVED** [review]

**Pull request merged**